UI/frontend-plugins/frontend-tools/scripts.cpp

```diff
@@ -470,6 +474,37 @@ void ScriptsTool::on_defaults_clicked()
 		item->data(Qt::UserRole).toString().toUtf8().constData());
 }
 
+void ScriptsTool::on_description_linkActivated(const QString &link)
+{
+	if (link.left(7).compare("http://", Qt::CaseInsensitive) == 0 ||
+	    link.left(8).compare("https://", Qt::CaseInsensitive) == 0) {
```

Comment:
Might be better to cast into QUrl and use scheme? 
 https://doc.qt.io/qt-5/qurl.html#scheme

Reply:
Hmm, might be worth doing. I simply copied the code we ended up using for browser source creation via drag and drop.

Reply:
Perfect. Done!